webui/.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "env": {
+      "browser": true,
+      "jest": true,
+      "es6": true
+    },
+    "settings": {
+      "react": {
+        "version": "17.0"
+      }
+    },
+    "rules": {
+      "eqeqeq": "warn",
```

Comment:
:-(

Reply:
It was a direct copy of what was defined in `package.json`. I don't like including these things in `package.json` because I feel it's a mixing of concerns (and `package.json` is over-loaded as is), so I moved it as-is to `.eslintrc`.
Having said that, I agree a `warn` is not enough for this rule. However, I need to check that it doesn't break existing code and refactor if it does.